analyzer-java/src/test/java/com/infosupport/ldoc/analyzerj/integritytest/SelfTest.java

```diff
@@ -0,0 +1,27 @@
+package com.infosupport.ldoc.analyzerj.integritytest;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.infosupport.ldoc.analyzerj.Main;
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+
+class SelfTest {
+
+  @Test
+  @SuppressWarnings("ResultOfMethodCallIgnored")
+  void testMain() throws IOException {
+    String outputJson = "result.json";
```

Comment:
Maybe we should give this a bit more specific name? Since result.json might be in use for other things, and this code deletes the file at the end.